_posts/2024-12-31-please-keep-your-information-up-to-date.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+published: True
+toc: true
+title: "Please keep your information up-to-date"
+date: 2024-12-31 11:00:00 +0000
+tags: authors contact cpan maintenance modules security
+author: Robert Rothenberg
+excerpt: "Some end of year reminders for CPAN Authors: Do all of your modules have up-to-date contact information?"
+---
+
+Some end of year reminders for CPAN Authors:
+
+Do all of your modules have up-to-date contact information? If not, please release new versions with an updated email address in the AUTHOR section.
+
+(And while you're at it, why not add a [security policy](https://security.metacpan.org/docs/guides/security-policy-for-authors.html) to your distribution, so that people know how and where to report a security issue with your module.)
+
+If you have a cpan.org email alias, does it forward to the correct email address?
```

Comment:
Add a link to the PAUSE page for editing the email setting, perhaps?  https://pause.perl.org/pause/authenquery?ACTION=edit_cred

Reply:
I'm not sure I link linking directly to that.